dvc/command/experiments/show.py

```diff
@@ -442,7 +443,8 @@ def show_experiments(
     row_styles = lmap(baseline_styler, td.column("typ"))
     for n, row_style in enumerate(row_styles):
         if n % 2 != 0:
-            row_style["style"] += " on grey23"
+            color = "grey85" if light else "grey23"
+            row_style["style"] += f" on {color}"
```

Comment:
This can just be

```py
        color = "black" if n % 2 == 0 else "bright_black"
        row_style["style"] += f" on {color}"
```

see https://github.com/iterative/dvc/pull/7202#issuecomment-1001848870